config.json

```diff
@@ -65,7 +65,7 @@
       "files": [
         {
           "destination": "colors.js",
-          "format": "javascript/umd",
+          "format": "javascript/es6",
```

Comment:
This basically changes the output so that variables can be used more directly. The other format was causing problems because some of the names had hyphens and It didn't like it. 

in UMD it looked like            
background2: { 
   lm: _styleDitcionary.color.background-2.lm.value,
  dm: color.background-2.dm.value
},

in ES6 it looks like
background2: {
    lm: ColorBackground2Lm,
    dm: ColorBackground2Dm
  },